packages/cactus-plugin-keychain-memory/README.md

```diff
@@ -1,9 +1,41 @@
 # `@hyperledger/cactus-plugin-keychain-memory`
 
-> TODO: description
+## Prometheus Exporter
 
-## Usage
+This class creates a prometheus exporter, which scraps the transactions (total transaction count) for the use cases incorporating the use of Fabric connector plugin.
 
+
+### Usage
+The prometheus exporter object is initialized in the `PluginKeychainMemory` class constructor itself, so instantiating the object of the `PluginKeychainMemory` class, gives access to the exporter object.
+You can also initialize the prometheus exporter object seperately and then pass it to the `IPluginKeychainMemoryOptions` interface for `PluginKeychainMemory` constructor.
+
+`getPrometheusExporterMetricsEndpointV1` function returns the prometheus exporter metrics, currently displaying the total key count, which currently increments everytime the `set()` and `delete()` method of the `PluginKeychainMemory` class is called.
```

Comment:
@jagpreetsinghsasan The function name does not have the `Endpoint` part in it (intentionally, so that's how it should be in the readme as well)

```typescript
const res = await apiClient.getPrometheusExporterMetricsV1();
```